fvm/transactionInvocator.go

```diff
@@ -180,6 +180,7 @@ func (i *TransactionInvocator) Process(
 		// if tx fails just do clean up
 		programs.Cleanup(nil)
 		i.logger.Info().
+			Err(txError).
```

Comment:
do we need this ?

Reply:
No, I can remove this.

Reply:
removed in https://github.com/onflow/flow-go/pull/295/commits/acc800ae6062ab84279f3910cdba750e6d28e3a6